extension/js/tree.js

```diff
@@ -0,0 +1,222 @@
+/*
+ *	DeepSite Source Tree page 
+*/
+
+if (!deepCite) {
+	var deepCite = {};
+}
+// DOM elements
+deepCite.citeComponent = document.getElementsByClassName("generated-cite-box")[0];
+deepCite.claimComponent = document.getElementsByClassName("generated-claim-box")[0];
+deepCite.treeContainer = document.getElementById("tree-container");
+deepCite.canvasElement = document.getElementById("canvas-element");
+
```

Comment:
for testData below, I appended two values to the current paradigm: citeID, and parentCiteID.  The tree page uses these two values in determining where it should place the cite records.  When we implement this, we will need some form of identification attributes, and I found these to be the easiest to implement.

Reply:
I implemented it so that id needs to be falsy for the original claim.  It wouldn't be hard to modify this, so just do the easiest thing server-side and the js can always be updated